### ISSUE TITLE
USER ISSUE: Left clicking to get items from distribution station crashes game

### ISSUE THREAD

**Issue by mrpeeps** [StrangeLoopGames/EcoIssues#12159]
**Version:** 0.8.1.4 beta

**Steps to Reproduce:**
Go to distribution station and left click

**Expected behavior:**
opened, or given me the items	

**Actual behavior:**

Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)</size>
```

**Do you have mods installed? Does issue happen when no mods are installed?:**

Server encountered an exception:

<size=60.00%>Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Components.StorageComponent.OnActLeft(InteractionContext context)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Interactions.InteractionExtensions.ExecuteUntilSuccess(IEnumerable`1 results)
   at Eco.Gameplay.Players.Player.PlayerInteract(InteractionInfo info)</size>
```

**Comment by beastyn** [comment]
@mrpeeps Thank you for your report!
Can you give your world save files including .db?
Here is an instruction https://github.com/StrangeLoopGames/EcoIssues/wiki/Uploading-a-world-save

Thank you!

**Comment by mrpeeps** [comment]
@beastyn Sorry, I forgot to mention that this isnt my server, its the public server "Old Redwood". Is it possible to get that data still?

**Comment by beastyn** [comment]
@mrpeeps Thats ok. Can you provide your client log?
https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file
I will try to reproduce it, so any additional info will be helpful )

**Comment by beastyn** [comment]
@mrpeeps It was easy to reproduce with old redwood. I will handle it from there, thank you again!

**Comment by beastyn** [comment]
Fixed in 8.2
